src/utils/response.js

```diff
@@ -0,0 +1,75 @@
+import boom from 'boom';
+import serializeError from 'serialize-error';
+// import Raven from 'raven';
+// import log from 'utils/log';
+
+// Raven.config(process.env.RAVEN_CONFIG, {
+//   release: '1.3.0'
+// }).install();
+
+function buildResponse(statusCode, data, message, isSuccess, noCache) {
+  const body = JSON.stringify({
+    data,
+    message,
+    success: isSuccess
+  });
+  const response = {
+    statusCode,
+    headers: {
+      'Access-Control-Allow-Origin': '*',
+      'Access-Control-Allow-Credentials': true
+    },
+    body
+  };
+  if (noCache) {
+    response.headers['Cache-Control'] = 'private, no-cache, no-store, must-revalidate';
+    response.headers['Expires'] = '-1'; // eslint-disable-line
+    response.headers['Pragma'] = 'no-cache'; // eslint-disable-line
+  }
+  return response;
+}
+
+export function success(body, isCreated, message, noCache) {
+  return buildResponse(isCreated ? 201 : 200, body, message, true, noCache);
+}
+
+function reportToRaven(err, event) {
+  if (event && event.requestContext && event.requestContext.authorizer && event.requestContext.authorizer.claims) {
+    delete event.requestContext.authorizer.claims['custom:pincode'];
+    // Raven.setContext({
+    //   user: event.requestContext.authorizer.claims
+    // });
+  }
+  return new Promise((resolve) => {
+    resolve();
+    // Raven.captureException(err, { extra: { err, event } }, (sendErr, eventId) => {
+    //   if (sendErr) {
+    //     // console.log('Raven Report Error');
+    //     // console.log(sendErr);
+    //   } else {
+    //     // console.log(`Raven Report id ${eventId}`);
+    //   }
+    //   resolve();
+    // });
+  });
+}
+
+export async function failure(err, event) {
+  await reportToRaven(err, event);
```

Comment:
I think you removed raven key from config, kindly add that back in and also add in serverless environments